docs/docs/user/installation.md

```diff
@@ -7,15 +9,23 @@ The Moog command-line-interface (CLI) can run on Linux and MacOS.
 The recommended way for end-users to install the Moog CLI is to download the
 pre-built binaries for your platform from the [releases page](https://github.com/cardano-foundation/moog/releases)
 
-For additional ways, see the CONTRIBUTING files in the [source repository](https://app.radicle.xyz/nodes/ash.radicle.garden/rad:z2a7Te5b28CX5YyPQ7ihrdG2EEUsC)
+Alternatively, if you are a Nix user you may run the CLI directly from the official repository:
```

Comment:
I think we said we would leave `nix` out of the user facing documentation. Alternatively, we could put it into some "note" or "info" box for "advanced" users?